edge-modules/api-proxy-module/src/monitors/token_client.rs

```diff
@@ -0,0 +1,70 @@
+use anyhow::{Error, Result};
+use log::info;
+use url::form_urlencoded::Serializer as UrlSerializer;
+
+pub const IOTHUB_ENCODE_SET: &percent_encoding::AsciiSet = &percent_encoding::CONTROLS // C0 control
```

Comment:
Can we use this instead? 
https://github.com/Azure/iotedge/blob/master/mqtt/edgelet-client/src/workload/client.rs#L15

Reply:
Fixed